### ISSUE TITLE
Will there be a 32bit version

### ISSUE THREAD

**Issue by Arthurgamepraix** [skyline-emu/skyline#271]
Because i wanted to download skyline but it did not install but my processor is a 64bit but is running in 32bit mode

**Comment by MrKing202981** [comment]
No, phones with 32 Bit SoCs/Operating System won't be able to run Skyline.
